out_writeapi.go

```diff
@@ -262,11 +270,13 @@ func rebuildPredicate(err error) bool {
 }
 
 // this function sends and checks the responses for data through a committed stream with exactly once functionality
-func sendRequestExactlyOnce(ctx context.Context, data [][]byte, config **outputConfig) error {
+func sendRequestExactlyOnce(ctx context.Context, data [][]byte, config **outputConfig, ind int) error {
```

Comment:
Perhaps use a more descriptive name for "ind"? Suggestion: streamIndex

Reply:
Done.